lib/methods/get-data.js

```diff
@@ -2,18 +2,17 @@ var merge = require('merge');
 
 module.exports = function() {
 
-  var data =  {
-    query:this.query,
-    limit:this.limit,
-    orderBy:this.orderBy.column,
-    ascending: this.orderBy.ascending,
-    page:this.page,
-    byColumn:this.options.filterByColumn?1:0
-  };
+  var data = {};
+  data[this.options.queryParams.query] = this.query;
+  data[this.options.queryParams.limit] = this.limit;
+  data[this.options.queryParams.orderBy] = this.orderBy.column;
+  data[this.options.queryParams.ascending] = this.orderBy.ascending;
+  data[this.options.queryParams.page] = this.page;
+  data[this.options.queryParams.byColumn] = this.options.filterByColumn?1:0;
 
   data = merge(data, this.options.params, this.customQueries);
 
   this.$dispatch('vue-tables.loading', data);
 
-  return this.$http.get(this.url, {params:data});
```

Comment:
Was it intended that all parameters are wrapped by "params[limit]" ( => `http://foo.bar/xyz/?params[limit]=10` )?
I removed this as it looked like wrong behaviour. Now its `http://foo.bar/xyz/?limit=10`